Verify profile with external applications (#502)
desmos-labs/desmos: docs/developers/msgs/profiles/unlink-application.md

# `MsgUnlinkApplication`

> This message allows you remove a previously linked application from your Desmos profile.

Reviewer: ```suggestion
This message allows you to remove a previously linked application from your Desmos profile.
```

## Structure

```json
{
  "@type": "/desmos.profiles.v1beta1.MsgUnlinkApplication",

>   "application": "<Name of the application to be unlinked>",

Reviewer: ```suggestion
  "application": "<Name of the application to unlink>",
```

  "username": "<Name of the account inside the application that should be unlinked>",
  "signer": "<Desmos address of the profile that should remove the link>"
}
```

### Attributes

| Attribute | Type | Description |
| :-------: | :----: | :-------- |

> | `application`  | `string` | Name of the application to be unlinked |

Reviewer: ```suggestion
| `application`  | `string` | Name of the application to unlink |
```
